server/src/main/java/org/opensearch/node/remotestore/RemoteStoreNodeAttribute.java

```diff
@@ -276,8 +276,6 @@ public boolean equals(Object o) {
 
     @Override
     public String toString() {
-        StringBuilder sb = new StringBuilder();
-        sb.append('{').append(this.repositoriesMetadata).append('}');
```

Comment:
I see @psychbot authored this. 

@psychbot - Was this string builder to be used in this method and it was a miss to not use it?

Reply:
Pretty sure it was a typo in the next line ( super -> sb )

Reply:
@dbwiddis I fixed the typo as per your suggestion